sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -629,31 +628,31 @@ public virtual Pageable<ConfigurationSetting> GetConfigurationSettings(SettingSe
         }
 
         /// <summary>
-        /// Retrieves the different revisions of a specific <see cref="ConfigurationSetting"/> with the specified <paramref name="key"/> and <paramref name="label"/>.
+        /// Retrieves the different revisions of one or more <see cref="ConfigurationSetting"/> entities that match the specified <paramref name="keyFilter"/> and <paramref name="labelFilter"/>.
         /// </summary>
-        /// <param name="key">The primary identifier of the configuration setting.</param>
-        /// <param name="label">A label used to group this configuration setting with others.</param>
+        /// <param name="keyFilter">The primary identifier of the configuration setting.</param>
```

Comment:
Let's describe this and labelFilter as filters used to select a subset of values in the configuration store, rather than using the descriptions for key and label.  For example, the label filter isn't used for groups (whereas the label is), it's used for selection.